src/chia_log/log_handler.py

```diff
@@ -48,26 +39,29 @@ def __init__(
         notify_manager: NotifyManager,
         stats_manager: Optional[StatsManager] = None,
     ):
+        self.services: Dict[EventService, List[Type[LogHandlerInterface]]] = {
+            EventService.HARVESTER: [HarvesterActivityHandler],
+            EventService.WALLET: [WalletAddedCoinHandler],
+            EventService.FULL_NODE: [BlockHandler, FinishedSignagePointHandler],
+            EventService.FARMER: [PartialHandler],
+        }
         self._notify_manager = notify_manager
         self._stats_manager = stats_manager
 
-        available_handlers: List[Type[LogHandlerInterface]] = [
-            HarvesterActivityHandler,
-            PartialHandler,
-            BlockHandler,
-            FinishedSignagePointHandler,
-            WalletAddedCoinHandler,
-        ]
-        self._handlers = []
-        for handler in available_handlers:
-            if _check_handler_enabled(config, handler.config_name()):
-                self._handlers.append(handler(config["handlers"][handler.config_name()]))
+        self._active_handlers = []
+        self._active_services = []
+        for service, service_handlers in self.services.items():
+            if service.name in config["monitored_services"].get(list):
+                self._active_services.append(service)
+                for handler in service_handlers:
+                    self._active_handlers.append(handler(config["handlers"][handler.config_name()]))
             else:
-                logging.info(f"Disabled handler: {handler.config_name()}")
-
+                logging.info(f"Disabled service monitoring: {service.name}")
+        # Init the keepalive monitor with enabled services
+        notify_manager._keep_alive_monitor.set_services(self._active_services)
```

Comment:
The keep-alive monitor already has access to the config, I don’t see a good reason for coupling the initialization of the monitor with the log handler. 🤔 

Can we fully initialize the KeepAliveMonitor from its own `__init__` method with 
```
self.set_services(config["monitored_services"].get(list))
```

Reply:
I'll have another go at it. This is what I initially tried but somehow it got tricky that way as well.

Reply:
Found an elegant-ish solution to the initial problem by expanding `EventServices` to have a string init. This has the added benefit of ignoring capitalization of the service names in the config.